webservice/tutorials/create-product-az.md

```diff
@@ -0,0 +1,476 @@
+        </description_short>
+    </product>
+</prestashop>
+```
+
+{{% notice note %}}
+Related request in the Postman collection: 
+
+7 - Update multi-lang fields
+{{% /notice %}}
+
+## Create a ProductType::TYPE_COMBINATIONS Product
+
+To create a Combination Product Type, you need to create your `Product` and then your `Combinations`. 
+
+### Create the Product
+
+Change in the xml body of the [Product creation request]({{< relref "#create-the-product" >}}), on the `<product_type>` node, the value from `standard` to `combinations`.
+
+Send a `POST` request to `/api/products` with this body: 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+    <product>
+        <id_manufacturer><![CDATA[1]]></id_manufacturer>
+        <id_supplier><![CDATA[1]]></id_supplier>
+        <id_category_default><![CDATA[1]]></id_category_default>
+        <new><![CDATA[1]]></new>
+        <id_default_combination><![CDATA[1]]></id_default_combination>
+        <id_tax_rules_group><![CDATA[1]]></id_tax_rules_group>
+        <type><![CDATA[1]]></type>
+        <id_shop_default><![CDATA[1]]></id_shop_default>
+        <reference><![CDATA[123456]]></reference>
+        <supplier_reference><![CDATA[ABCDEF]]></supplier_reference>
+        <ean13><![CDATA[1231231231231]]></ean13>
+        <state><![CDATA[1]]></state>
+        <product_type><![CDATA[combinations]]></product_type>
+        <price><![CDATA[123.45]]></price>
+        <unit_price><![CDATA[123.45]]></unit_price>
+        <active><![CDATA[1]]></active>
+        <meta_description>
+            <language id="1"><![CDATA[Description]]></language>
+        </meta_description>
+        <meta_keywords>
+            <language id="1"><![CDATA[Keywords]]></language>
+        </meta_keywords>
+        <meta_title>
+            <language id="1"><![CDATA[My Title for SEO]]></language>
+        </meta_title>
+        <link_rewrite>
+            <language id="1"><![CDATA[awesome-product]]></language>
+        </link_rewrite>
+        <name>
+            <language id="1"><![CDATA[My awesome Product with combinations]]></language>
+        </name>
+        <description>
+            <language id="1"><![CDATA[Description]]></language>
+        </description>
+        <description_short>
+            <language id="1"><![CDATA[Short description]]></language>
+        </description_short>
+        <associations>
+            <categories>
+                <category>
+                    <id><![CDATA[1]]></id>
+                </category>
+            </categories>
+        </associations>
+    </product>
+</prestashop>
+```
+
+{{% notice note %}}
+Related request in the Postman collection: 
+
+8 - Create Combination Product
+{{% /notice %}}
+
+### Create the Combinations
+
+Send two `POST` requests to `/api/combinations` to create your combinations, with `{{id_product}}` your previously created product id. 
```

Comment:
```suggestion
Send two `POST` requests to `/api/combinations` to create your combinations. Use `{{id_product}}` of your previously created `Product`. 
```

Why do we need to send it twice? Also, it seems there's no information about the combination, like... which group of the attribute is thta? what's the value?

Reply:
cc @thomasnares

Reply:
yes i still need to change this paragraph and explain about attributes 👍

Reply:
ok, I converted your PR to draft :D let me know once it's ready